ui/page.go

```diff
@@ -630,6 +639,72 @@ func (page pageCommon) SelectedAccountLayout(gtx layout.Context) layout.Dimensio
 	return decredmaterial.Card{}.Layout(gtx, selectedDetails)
 }
 
+type SubPage struct {
+	title      string
+	walletName string
+	back       func()
+	body       layout.Widget
+}
+
+func (page pageCommon) SubPageLayout(gtx layout.Context, sp SubPage) layout.Dimensions {
+	if page.subPageInfoButton.Button.Clicked() {
+		go func() {
+			page.modalReceiver <- &modalLoad{
+				template:   InfoTemplate,
+				title:      "Verify message",
+				cancel:     page.closeModal,
+				cancelText: "Got it",
+			}
+		}()
+	}
+
+	if page.subPageBackButton.Button.Clicked() {
+		sp.back()
+	}
+
+	return decredmaterial.Card{Color: page.theme.Color.Surface, Rounded: true}.Layout(gtx, func(gtx C) D {
```

Comment:
kindly rebase as the Card widget has been updated. `Color` and `Rounded` params no longer exist in card struct.